.github/workflows/ci.yml

```
@@ -1001,14 +1046,15 @@ jobs:
     needs:
       - setup
 
-      # Basic
+      # Platforms
       - build_all
       - build_test_all_windows
       - build_test_all_macos_arm64
       - build_test_all_macos_x86_64
       - build_test_all_bazel
       - test_all
       - test_gpu
+      - test_nvidia_sm80
```

"Platforms" doesn't feel right for "build_all" and "test_all", at least. How about this?

```suggestion
      # Basic
      - build_all
      - test_all
      - build_test_all_bazel
      
      # Platforms
      - build_test_all_windows
      - build_test_all_macos_arm64
      - build_test_all_macos_x86_64
      
      # Accelerators
      - test_gpu
      - test_nvidia_sm80
```

---

I was thinking it's the default platform :grin: Sure that organization SGTM

---

You suggested trailing spaces :angry:

---

I guess we'll rerun the whole CI just to fix those now :stuck_out_tongue: